previews/primer/beta/flash_preview/with_action.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= render(Primer::Beta::Flash.new(full: full, dismissible: dismissible, icon: icon, scheme: scheme)) do |component| %>
+  <% component.with_action(size: :small) { "Take action" } %>
+  <%= content %>
+<% end %>
```

Comment:
Not sure if it's meant to be this way, but the button alignment felt off here for a single line 
<img width="550" alt="image" src="https://user-images.githubusercontent.com/54012/195734691-64507aca-c046-4833-845d-15cb589af25c.png">

Reply:
This felt off to me as well. I center aligned it.